api/src/pcapi/utils/requests.py

```diff
@@ -105,3 +107,6 @@ def __init__(self, *args, disable_synchronous_retry: bool = False, **kwargs):  #
         self.mount("https://www.demarches-simplifiees.fr", safe_adapter)
         self.mount(settings.UBBLE_API_URL, safe_adapter)
         self.mount(settings.BATCH_API_URL, unsafe_adapter)
+
+    def request(self, method: Union[str, bytes], url: Union[str, bytes], *args: Any, **kwargs: Any) -> Response:
```

Comment:
`typing.Union` c'est tellement 2021 (Python 3.9). Tu vis dans le passé ! :)

```suggestion
    def request(self, method: str | bytes, url: str | bytes, *args: Any, **kwargs: Any) -> Response:
```

Idem plus haut.

Au fait (par curiosité), pourquoi change-t-on le typage de `method` et `url` : c'est pour être raccord avec le typage de `requests` ? En local,

Reply:
oui c'est pour être raccord avec la signateure de Session (mypy râlait)